packages/stylelint-config/package.json

```diff
@@ -8,12 +8,12 @@
   "main": "index.js",
   "license": "MIT",
   "peerDependencies": {
-    "stylelint": "^12.0.0"
+    "stylelint": "^12.0.0 || ^13.0.0"
```

Comment:
We are really support both version ? I think we shoudl force use of last version :)

Reply:
It was just because all dependencies support both versions. But yes, we just can support the last one.